### PR TITLE
add doc strings

### PR DESCRIPTION
**Pull request by Majid-justice**
https://tools.hmcts.net/jira/browse/FACT-1266

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by amstevenson** [comment]
Do we want doc strings on all methods at the service level as well? And for each, do we want to provide information about parameters too?

File: src/main/controllers/service/ChooseServiceAreaController.ts [hunk 1]
```diff
@@ -15,7 +15,11 @@ export class ChooseServiceAreaController {
     private readonly api: FactApi,
     private readonly serviceAreaRedirect: ServiceAreaRedirect
   ) { }
-
+  /**
+   * GET /services/:serviceChosen/service-areas/:action
+   * gets and returns the service area data
+   * @params serviceChosen string, action string
```
[the remark this answers is not in the document]
Comment (Majid-justice) [reply src/main/controllers/service/ChooseServiceAreaController.ts]: update as per your suggestion

File: src/main/controllers/service/ServicePostcodeResultsController.ts [hunk 2]
```diff
@@ -40,7 +44,12 @@ export class ServicePostcodeResultsController {
       .replace('{postcode}', postcode);
     return res.render('postcode-results', data);
   }
-
+  /**
+   * GET /get
+   * redirects to the service results page
+   * @params serviceArea string, action string
```
[the remark this answers is not in the document]
Comment (Majid-justice) [reply src/main/controllers/service/ServicePostcodeResultsController.ts]: update as per your suggestion

File: src/main/controllers/ChooseActionController.ts [hunk 3]
```diff
@@ -5,11 +5,18 @@ import { PageData } from '../interfaces/PageData';
...
+   * @returns redirects to the choose action page.
+   */
   public get(req: FactRequest, res: Response): void {
     res.render('choose-action', req.i18n.getDataByLanguage(req.lng)['choose-action']);
   }
-
+  /**
+   * POST /post
+   * @returns redirects to the service page for the selected action
+   * @param {string} req.body.chooseAction
```
[the remark this answers is not in the document]
Comment (Majid-justice) [reply src/main/controllers/ChooseActionController.ts]: update as per your suggestion

File: src/main/controllers/service/ChooseServiceController.ts [hunk 4]
```diff
@@ -12,7 +12,11 @@ export class ChooseServiceController {
   constructor(
     private readonly api: FactApi
   ) { }
-
+  /**
+   * GET /services
+   * @returns gets the service area data.
+   * @param {string} action
```
[the remark this answers is not in the document]
Comment (Majid-justice) [reply src/main/controllers/service/ChooseServiceController.ts]: update as per your suggestion

File: src/main/controllers/service/ServiceAreaRedirect.ts [hunk 5]
```diff
@@ -10,7 +10,13 @@ export class ServiceAreaRedirect {
     [Action.Update]:        [Catchment.National, Catchment.Regional, Catchment.Local],
     [Action.NotListed]:     [Catchment.National, Catchment.Regional, Catchment.Local],
   };
-
+  /**
+   * GET /getUrl/:service/:serviceArea/:action
+   * @returns redirect a service area to the correct page.
+   * @param {string} service
+   * @param {ServiceAreaResult}serviceArea
```
[the remark this answers is not in the document]
Comment (Majid-justice) [reply src/main/controllers/service/ServiceAreaRedirect.ts]: update as per your suggestion

File: src/main/controllers/service/ServicePostcodeResultsController.ts [hunk 6]
```diff
@@ -12,7 +12,11 @@ export class ServicePostcodeResultsController {
   constructor(
     private readonly api: FactApi
   ) { }
-
+  /**
+   * GET /getCourtResultsByPostcode
+   * @returns redirect to postcode service results page.
+   * @param {string} req.query.postcode
```
[the remark this answers is not in the document]
Comment (Majid-justice) [reply src/main/controllers/service/ServicePostcodeResultsController.ts]: update as per your suggestion

**Review by justiceia — APPROVED** [review]

**Pull request merged**